tsl/test/expected/dist_policy.out

```diff
@@ -0,0 +1,161 @@
+-- This file and its contents are licensed under the Timescale License.
+-- Please see the included NOTICE for copyright information and
+-- LICENSE-TIMESCALE for a copy of the license.
+-- Need to be super user to create extension and add data nodes
+\c :TEST_DBNAME :ROLE_CLUSTER_SUPERUSER;
+\unset ECHO
+psql:include/remote_exec.sql:5: NOTICE:  schema "test" already exists, skipping
+CREATE OR REPLACE FUNCTION test_drop_chunks(job_id INTEGER)
+RETURNS VOID
+AS :TSL_MODULE_PATHNAME, 'ts_test_auto_drop_chunks'
+LANGUAGE C VOLATILE STRICT;
```

Comment:
@svenklemm You commented previously to use new API instead of such functions: https://github.com/timescale/timescaledb/pull/2192#pullrequestreview-461552940
Is it something to consider here for @mkindahl ?

Reply:
atm it requires a bit effort because calling the procedure directly requires you to have the config jsonb object so i'm fine leaving it like that for now but once we have `run_job` all those `ts_test_auto_' helper functions for policies will be removed

Reply:
@svenklemm Thank you for clarifying!